### PR TITLE
Microsoft.StorageSync - Fix Issue 4559 and other known problems with preview version

### PR DESCRIPTION
**Pull request by ankushbindlish2**
Issue : https://github.com/Azure/azure-rest-api-specs/issues/4559#event-1987241659

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.

### CODE REVIEW

File: specification/storagesync/resource-manager/Microsoft.StorageSync/preview/2018-10-01/storagesync.json [hunk 1]
```diff
@@ -3342,7 +3342,112 @@
...
+                }
+            }
+        },
+        "SyncProgressStatus": {
+            "type": "object",
+            "description": "Sync Session status object.",
+            "properties": {
+                "progressTimestamp": {
+                    "type" : "string",
+                    "description": "progress timestamp"
```
Comment (nschonni) [inline specification/storagesync/resource-manager/Microsoft.StorageSync/preview/2018-10-01/storagesync.json]: Progress capitalization?

File: specification/storagesync/resource-manager/Microsoft.StorageSync/preview/2018-10-01/storagesync.json [hunk 2]
```diff
@@ -3342,7 +3342,112 @@
...
+                    "maximum": 100,
+                    "description": "Last sync status"
+                },
+                 "lastSyncTimestamp": {
+                    "type" : "string",
+                    "description": "Last sync timestamp"
+                },
+                 "lastSyncSuccessTimestamp": {
+                     "type" : "string",
+                    "description": "Last sync access timestamp"
```
Comment (nschonni) [inline specification/storagesync/resource-manager/Microsoft.StorageSync/preview/2018-10-01/storagesync.json]: access -> success

**Comment by ankushbindlish2** [comment]
@nschonni  Can you help me with conflicting files

File: specification/storagesync/resource-manager/Microsoft.StorageSync/preview/2018-10-01/storagesync.json [hunk 3]
```diff
@@ -3397,7 +3397,7 @@
                 },
                  "lastSyncSuccessTimestamp": {
                      "type" : "string",
-                    "description": "Last sync access timestamp"
+                    "description": "Last sync sucess timestamp"
```
Comment (nschonni) [inline specification/storagesync/resource-manager/Microsoft.StorageSync/preview/2018-10-01/storagesync.json]: Missing a "c" in "success"

**Comment by nschonni** [comment]
@ankushbindlish2 it looks like you're the last one to edit the file https://github.com/Azure/azure-rest-api-specs/blob/master/specification/storagesync/resource-manager/Microsoft.StorageSync/preview/2018-10-01/storagesync.json
Maybe you made the changes off an older branch?
Usually I'd suggest `git pull --rebase upstream master` and force push the branch, but I'm not sure what the process is at Microsoft

**Comment by ankushbindlish2** [comment]
thanks !

**Pull request closed without merging**